### ISSUE TITLE
Rename error when file name has special characters

### ISSUE THREAD

**Issue by sunlewuyou** [pldubouilh/gossa#40]
See log：
```
2020/02/20 12:32:01 error [rpc {mv [/油管_韩雪&amp;鞠婧祎&amp;原版三版《红昭愿》放在一起自见分晓.mp4 /韩雪&amp;鞠婧祎&amp;原版三版《红昭愿》放在一起自见分晓.mp4]}] rename /root/share/油管_韩雪&amp;鞠婧祎&amp;原版三版《红昭愿》放在一起自见分晓.mp4 /root/share/韩雪&amp;鞠婧祎&amp;原版三版《红昭愿》放在一起自见分晓.mp4: no such file or directory
```

Original file name：
油管_韩雪&鞠婧祎&原版三版《红昭愿》放在一起自见分晓.mp4

**Comment by pldubouilh** [comment]
indeed, it seems the `&` wasn't properly escaped, of all things :) 
I have a fix, I'll test it for a few days and push it out.

**Comment by sunlewuyou** [comment]
Thank you for your timely repair, this project will definitely help many people better!

**Comment by pldubouilh** [comment]
should be fixed, check [v0.1.2](https://github.com/pldubouilh/gossa/releases/tag/v0.1.2)

**Comment by sunlewuyou** [comment]
The test is normal, thank you!
